seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/connection/SimpleJdbcConnectionProvider.java

```diff
@@ -121,24 +121,29 @@ public Connection getOrEstablishConnection()
         if (jdbcOptions.getPassword().isPresent()) {
             info.setProperty("password", jdbcOptions.getPassword().get());
         }
-        connection = driver.connect(jdbcOptions.getUrl(), info);
+        String jdbcOptionsUrl = jdbcOptions.getUrl();
+        connection = driver.connect(jdbcOptionsUrl, info);
         if (connection == null) {
             // Throw same exception as DriverManager.getConnection when no driver found to match
             // caller expectation.
             throw new SQLException(
-                "No suitable driver found for " + jdbcOptions.getUrl(), "08001");
+                "No suitable driver found for " + jdbcOptionsUrl, "08001");
         }
 
-        //Auto commit is used by default
-        connection.setAutoCommit(true);
-
+        // Auto commit is used by default, Teradata jdbc type fastload must be set false.
+        connection.setAutoCommit(!(jdbcOptionsUrl.toLowerCase().contains("jdbc:teradata:") & jdbcOptionsUrl.toLowerCase().contains("type=fastload")));
```

Comment:
cc @ic4y

Reply:
Waiting for the #3451 pr to be merged, there is no need to modify it.